src/node_crypto.cc

```diff
@@ -1202,6 +1202,8 @@ void SecureContext::SetFreeListLength(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+// Currently, EnableTicketKeyCallback and TicketKeyCallback are only present for
+// the regression test in test/parallel/test-https-resume-after-renew.js.
```

Comment:
I think it’s implied that they are private when they are only there for tests. Also, this is only on the `_handle` property, so I think we’re good.